RELEASE.md

```diff
@@ -6,14 +6,36 @@ Creating a new release of network proxy involves releasing a new version of the
 
 1. The first step involves creating a new git tag for the release, following semvar for go libraries. A tag is required for both the repository and the konnectivity-client library. For example releasing the `0.0.15` version will have two tags `v0.0.15` and `konnectivity-client/v0.0.15` on the appropriate commit.
 
-    The exact commands are
+    Decide the next tag. Currently 0.1 minor version uses `master` branch, and 0.0 minor version uses `release-0.0` branch.
```

Comment:
I think we should specify that we expect the 'master' branch to be that latest version, which is currently 0.1.

Reply:
PTAL